api/migrations/add_slug_to_not_null.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return Promise.all([
+      queryInterface.sequelize.query(`
```

Comment:
They should not be running in parallel.

Reply:
I think to update single column, multiple queries are needed so they should run in parallel and so therefore promiseall is also needed.

Reply:
You can do all that in a single query.

Reply:
i think to update multiple things in a single column, we need to execute multiple queries, it can't be performed in a single query but if it is possible then can you can share some resource link so i can study about it. It will be highly appreciated, thanks

Reply:
You can do in a single query, try to cascade.